src/com/epam/homework/sets/Node.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.homework.sets;
+
+public class Node<T extends Comparable<T>> {
```

Comment:
Node can be a nested class inside your Set

Reply:
Should I change it and make a nested class?

Reply:
You may change it

Reply:
I changed Node to inner class and pushed